### PR TITLE
Initialise RNAsum v0.5.0

### PR DESCRIPTION
**Pull request by skanwal**
- Tested on ICA with `wfr.0e2dbd7255114e109914eb7cad7e82b7`.

### CODE REVIEW

File: tools/rnasum/0.5.0/rnasum__0.5.0.cwl [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+      prefix: "--cn_loss"
+  cn_gain:
+    label: cn gain
+    default: 95
+    doc: |
+      CN threshold value to classify genes within gained regions.
+    type: int?
+    inputBinding:
+      prefix: "--cn_gain"
+  clinical_info:
```
Comment (pdiakumis) [inline tools/rnasum/0.5.0/rnasum__0.5.0.cwl:179]: We decided to get rid of this one

File: tools/rnasum/0.5.0/rnasum__0.5.0.cwl [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+    label: clinical info
+    doc: |
+      xslx file with clinical information.
+    type: File?
+    inputBinding:
+      prefix: "--clinical_info"
+  clinical_id:
```
Comment (pdiakumis) [inline tools/rnasum/0.5.0/rnasum__0.5.0.cwl:186]: Ditto

File: tools/rnasum/0.5.0/rnasum__0.5.0.cwl [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+      prefix: "--project"
+  top_genes:
+    label: top genes
+    default: 5
+    doc: |
+      The number of top ranked genes to be presented.
+    type: int?
+    inputBinding:
+      prefix: "--top_genes"
+  hide_code_btn:
```
Comment (pdiakumis) [inline tools/rnasum/0.5.0/rnasum__0.5.0.cwl:231]: Ditto

File: tools/rnasum/0.5.0/rnasum__0.5.0.cwl [hunk 4]
```diff
@@ -0,0 +1,277 @@
...
+    label: hide code btn
+    doc: |
+      Hide the "Code" button allowing to show/hide code chunks in the final HTML report.
+    type: boolean?
+    inputBinding:
+      prefix: "--hide_code_btn"
+  grch_version:
```
Comment (pdiakumis) [inline tools/rnasum/0.5.0/rnasum__0.5.0.cwl:238]: Ditto

**Review by pdiakumis — APPROVED** [review]
Looks good! Just left a couple comments about deprecated CLI options, but you can ignore those. I'll work on adding some of the other options (e.g. `--arriba_pdf`,  `--dragen_mapping_metrics`, `--dragen_fusions`) (and destroying everything!) while you're away :-)

File: tools/rnasum/0.5.0/rnasum__0.5.0.cwl [hunk 5]
```diff
@@ -0,0 +1,277 @@
...
+    listing:
+      - entryname: $(get_script_path())
+        entry: |
+          #!/usr/bin/bash
+
+          # Fail on non-zero exit code
+          set -euo pipefail
+
+          # Run rnasum with input parameters
+          $(get_eval_line())
```
Comment (alexiswl) [inline tools/rnasum/0.5.0/rnasum__0.5.0.cwl:51]: Could be replaced with

```yaml
requirements: 
  InlineJavascriptRequirement: {}


baseCommand: [ "rnasum.R" ]

arguments: 
  - "--batch_rm"
  - "--filter" 
  - "--log" 
  - "--save_tables" 
  - "--pcgr_splice_vars"
```

**Comment by pdiakumis** [comment]
Why did that Action get stuck...
Think it hit a time limit?

**Comment by alexiswl** [comment]
> Why did that Action get stuck...
> Think it hit a time limit?

Very strange, haven't seen that one in a while. Will debug that today.

**Comment by alexiswl** [comment]
Might be related to https://github.com/umccr/cwl-ica/pull/419/files

**Comment by alexiswl** [comment]
Waiting on 

* https://github.com/umccr/infrastructure/pull/387
* https://github.com/umccr/cwl-ica/pull/448

**Comment by victorskl** [comment]
Do we want to backport this to Ocicat?

**Comment by pdiakumis** [comment]
> Do we want to backport this to Ocicat?
No, we need to test this and refurbish the interiors a fair bit first ;-)
I'll check your PRs after 4pm Alexis, no rush with this one.

**Pull request merged**